independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/Methods.java

```diff
@@ -68,7 +68,7 @@ static void addDelegatingMethods(IndexView index, ClassInfo classInfo, Map<TypeV
                     Type returnType = key.method.returnType();
                     Type[] params = new Type[key.method.parameters().size()];
                     for (int i = 0; i < params.length; i++) {
-                        params[i] = resolveType(key.method.parameters().get(i), resolvedTypeParameters);
```

Comment:
I wonder if we actually need the `resolvedTypeParameters` in this method. I think we don't.